apps/prairielearn/assets/scripts/lib/questionsTable.ts

```diff
@@ -255,6 +256,28 @@ onDocumentReady(() => {
 
   $('#questionsTable').bootstrapTable(tableSettings);
 
+  const startFromInput = document.querySelector<HTMLInputElement>('#start_from');
+  // The startFromInput either has value 'Template' or 'Empty question'
```

Comment:
Nit, conventionally comments should come above the line that they describe. Ditto elsewhere in this PR.